examples/theano/robo_cartpole.py

```diff
@@ -0,0 +1,32 @@
+import gym, roboschool
+import numpy as np
+import random
+
+def demo_run():
+    env = gym.make("RoboschoolInvertedPendulum-v1")
+
+    while True:
+        frame = 0
+        score = 0
+        restart_delay = 0
+        obs = env.reset()
+        print("reseted")
+        while True:
+            a = env.action_space.sample()
+            obs, r, done, _ = env.step(a)
+            score += r
+            frame += 1
+            still_open = env.render("human")
+            if still_open==False:
+                return
+            if not done: continue
+            if restart_delay==0:
+                print("score=%0.2f in %i frames" % (score, frame))
+                restart_delay = 60*2  # 2 sec at 60 fps
+            else:
+                restart_delay -= 1
+                if restart_delay==0: break
+
+
+if __name__=="__main__":
```

Comment:
Please write these in the style of a regular garage launcher rather than a Python script.